src/java/org/apache/poi/ss/usermodel/Row.java

```diff
@@ -50,6 +50,28 @@ Licensed to the Apache Software Foundation (ASF) under one or more
      * @return Cell a high level representation of the created cell.
      * @throws IllegalArgumentException if columnIndex &lt; 0 or greater than a maximum number of supported columns
      * (255 for *.xls, 1048576 for *.xlsx)
+     * @see CellType#BLANK
+     * @see CellType#BOOLEAN
+     * @see CellType#ERROR
+     * @see CellType#FORMULA
+     * @see CellType#NUMERIC
+     * @see CellType#STRING
+     * @deprecated POI 3.15 beta 3. Use {@link #createCell(int, CellType)} instead.
+     */
+    Cell createCell(int column, int type);
```

Comment:
This was removed because we no longer want to pass cell type as an int - why do you want to add it back?

Reply:
In fact I do not. I used a bit older version as a base for my version, and this was still in charge. 
Removing now, from all *Row classes.